doc/release/1.13.0-notes.rst

```diff
@@ -137,6 +137,11 @@ invokes``ndarray.__getslice__`` (e.g. through ``super(...).__getslice__``) will
 now issue a DeprecationWarning - ``.__getitem__(slice(start, end))`` should be
 used instead.
 
+Indexing MaskedArrays/Constants with ... (``ellipsis``) now returns MaskedArray
```

Comment:
That should be `Ellipsis`, not `ellipsis`. I think @charris' comment was requesting ` ``...`` `